portfolio/src/main/webapp/charts-script.js

```diff
@@ -0,0 +1,24 @@
+google.charts.load('current', {
+  'packages':['geochart'], 
+  'mapsApiKey': 'AIzaSyARIbGzg1ObeVicxm6Txr1mqhUR4w12LQQ'
```

Comment:
Have you restricted this maps API key to only be used from this website?  In general I think we want to try to avoid publishing api keys on github for security reasons.

Reply:
Yes, I restricted the API key. GitHub also sent me an email about this yesterday -- is there a way to use the key without directly adding it to the code?